src/shared/utils.ts

```diff
@@ -113,6 +113,10 @@ export class Utils {
     return metaData;
   }
 
+  public static prettyPrint(json) {
```

Comment:
Pretty :)

Reply:
@mydiemho initially added this in her PR.